File: src/main/java/br/com/correios/enderecador/cep/dnec/DnecStrategyCEP.java
package br.com.correios.enderecador.cep.dnec;

import br.com.correios.enderecador.cep.CEPFactory;
import br.com.correios.enderecador.cep.StrategyCEP;

public class DnecStrategyCEP implements StrategyCEP {
    public CEPFactory getFactory() {
        return CEPFactory.factory(1);
    }
}

File: src/main/kotlin/br/com/correios/enderecador/cep/dnec/DnecStrategyCEP.kt
package br.com.correios.enderecador.cep.dnec

import br.com.correios.enderecador.cep.CEPFactory
import br.com.correios.enderecador.cep.StrategyCEP

class DnecStrategyCEP : StrategyCEP {
    override val factory: CEPFactory?
        get() = CEPFactory.factory(1)
}
